ethereum-consensus/src/deneb/polynomial_commitments.rs

```diff
@@ -0,0 +1,138 @@
+#![allow(unused)]
+use crate::{crypto::hash, ssz::ByteVector};
+use alloy_primitives::{uint, U256};
+use blst::min_pk::PublicKey;
```

Comment:
why is this here?

imo you'll save a lot of time/confusion if you _don't_ develop with things like `#![allow(unused)]`

Reply:
i've just got a lot of warnings that are irrelevant